docs/README.md

```diff
@@ -13,17 +13,18 @@ If you plan to build a webapp to run on Cozy, you'll probably want to use a simp
 
 ## CSS Styleguide
 
-Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all the variables, mixins, classes, utilities and how to use them.
+Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all the variables, mixins, classes, utilities and how to use them with CSS classes only
 
 ## React components
 
-Check our [UI components](https://cozy.github.io/cozy-ui/react/) to see how to use ready made components.
+Check our [UI components](https://cozy.github.io/cozy-ui/react/) to see how to use ready made React components.
 
 ## Usage
 
+### As a Stylus plugin
 [Stylus][stylus] is used as a preprocessor. You can add it as a library in your project to use it out-of-the-box.
 
-Cozy UI is distributed as a [Stylus][stylus] plugin.
+Cozy UI can be distributed as a [Stylus][stylus] plugin
```

Comment:
Are we sure we want to communicate around stylus? We will be locked with stylus.

Reply:
So, just "as a plugin" ?